src/styles/components/_search-box.scss

```diff
@@ -0,0 +1,39 @@
+.search-box__input-container {
+  width: 60%;
+  margin: auto;
+}
+
+.search-box__artist-input-wrapper {
```

Comment:
Wydaje mi się, że możesz połączyć to z `input-container` i pozbyć się dodatkowego `span`